JenkinsfileRT_dev

```diff
@@ -74,12 +74,11 @@ bc0.conda_ver = '22.11.1'
 bc0.conda_packages = [
     "python=${python_version}",
 ]
-bc0.pip_reqs_files = ['requirements-dev.txt']
+bc0.pip_reqs_files = ['requirements-dev-st.txt', 'requirements-dev-st.txt']
```

Comment:
Is `bc0.pip_reqs_files` supposed to contain the same item twice?

Reply:
Good catch!